### DISCUSSION TITLE
How does GitHub help with Open Source Compliance?

### DISCUSSION THREAD

**Discussion by AjayDhage** [github/india#49] [Open Source] [answered]
We have been looking at a lot of features that GitHub provides for Security, Workflows etc. But does it have any tools, features, workflows that also help with the Open Source Compliance?

**Comment by karthiksedoc** [comment, marked as answer]
The details are here https://github.com/security/trust for you

**Reply by AjayDhage** [reply]
This is helpful! Could you please give more information on the "Governance Risk and Compliance" in the https://github.com/security/team

**Reply by karthiksedoc** [reply]
Feel free to reach the team [here](https://support.github.com/contact) for more information
